benchmarks/python/global_params.py

```diff
@@ -73,7 +73,7 @@ def generate_input_sizes(dims: Union[int, List] = 2) -> List[Tuple]:
         if dim == 2:
             input_ranges = []
 
-            step_size = 256
+            step_size = 1024
```

Comment:
IIUC, this step size means the problem sizes would be `x + 1024 * i`, where `x` is the minimum size and `i` is increased from 0 until `x + 1024 * i` reaches the maximum size. Is that correct?

Another way of generating problem sizes would be `x + some_factor ^ i`, e.g., `2, 4, 8, 16, 32, ...`.